services/brig/deb/opt/brig/new-version

```diff
@@ -0,0 +1 @@
+1.0.28
```

Comment:
@lipis Can we name this something other than just `new-version`. Maybe `templates-version`?

Reply:
Sure